src/utils/Firebase.tsx

```diff
@@ -91,3 +92,26 @@ export async function getUserJobs(userId: string): Promise<string[]> {
   // console.log(`Job information: ${jobInfos}`); // Debug log
   return jobInfos.filter(info => info !== null);
 }
+
+export async function createNewJob(userID: string, title: string, description: string, jobSkills: string[], header: string, thumbnail: string, jobDuration: number, questions: string[]) {
+  try {
```

Comment:
Should add validation for important values, such as ID, title, and description, although will have validation on front end functions, best to have a fail safe just incase. 

if (!userID || !title) {
console.error (error);
return;
}